Assets/Scripts/Data/Pickup Pup/PPGameSave.cs

```diff
@@ -24,14 +32,53 @@ public CurrencySystem Currencies
 
 	#endregion
 
-    CurrencySystem currencies;
-
-	public PPGameSave(DogDescriptor[] dogs, CurrencySystem currencies) 
+	public PPGameSave(DogDescriptor[] adoptedDogs, DogDescriptor[] scoutingDogs, CurrencySystem currencies) 
 	{
-		this.AdoptedDogs = new List<DogDescriptor>(dogs);
+		this.AdoptedDogs = new List<DogDescriptor>(adoptedDogs);
+		this.ScoutingDogs = new List<DogDescriptor>(scoutingDogs);
```

Comment:
Perhaps move in to `DogDatabse`? Or make a new class: `DogStorageSystem`? Either way, will hold off un such refactors until after this is merged and we're past first playable